src/components/delete_account/index.js

```diff
@@ -61,9 +61,10 @@ class DeleteAccount extends React.Component {
         {this.props.deleting ? (
           <p>{this.t('deleted')}</p>
         ) : (
-          <IrmaButton theme={'secondary'} onClick={() => this.onDelete()}>
+          <YiviButton theme={'ghost'} onClick={() => this.onDelete()}>
+            <CrossIcon />
             {this.t('delete')}
-          </IrmaButton>
+          </YiviButton>
```

Comment:
More a design remark, in my opinion the 'Delete MyYivi' button looks a bit small compared to the large white buttons.
![image](https://user-images.githubusercontent.com/17035191/225922542-2e5c671b-c917-4c53-a5de-c0a9071323db.png)

Reply:
Good point, let's ask Rosanne